core/gesture.ts

```diff
@@ -357,6 +357,16 @@ export class Gesture {
     }
   }
 
+  private getSelectedBlock(): BlockSvg | null {
```

Comment:
Can you add some docs explaining the purpose of this please

Reply:
This has been removed in the latest changes.